opentech/apply/funds/admin.py

```diff
@@ -75,6 +98,7 @@ class ApplyAdminGroup(ModelAdminGroup):
     menu_icon = 'folder-open-inverse'
     items = (
         RoundAdmin,
+        ScreeningStatusAdmin,
```

Comment:
I think putting this lower down the order is more reasonable. Before or after `CategoryAdmin` i think

Reply:
I have done this.